src/i18n/locales/en.json

```diff
@@ -1299,7 +1301,7 @@
a constitui „spam” în conformitate cu orice lege sau regulament aplicabil este expres interzisă. Ease își rezervă dreptul de a dezactiva orice Coduri personale distribuite prin canale neautorizate și de a revoca toate Recompensele de Recomandare.\n\nEase își rezervă dreptul, la propria sa discreţie, de a dezactiva orice Link Personal (și anula toate Recompense de Recomandare) pe care le consideră, la propria sa discreţie, că sunt utilizate cu încălcarea acestor Termeni.\n\nAchiziţiile de către prietenii referiţi care au efectuat anterior o achiziţie de la Ease sub orice nume, cont, adresă de e-mail sau alias nu sunt contabile ca achiziţii calificate și nu vă vor ajuta să câștigaţi recompense de recomandare. Orice recompense de trimitere emise în circumstanţele în care ar fi invalide în conformitate cu acești Termeni sunt nule.\n\nRecompensele de recomandare sunt de natură promoţională, nu sunt transferabile, nu au valoare în numerar și nu pot fi răscumpărate, în niciun caz, pentru carduri cadou, numerar sau orice echivalent în numerar. Fără a limita generalitatea celor menţionate mai sus, Recompensele de recomandare nu pot fi combinate între participanţi sau transmise către nimeni.\n\n4. TERMENI SUPLIMENTARI\nOrice Recompense de recomandare care nu pot fi verificate pentru satisfacţia Ease sunt supuse ajustării, anulării, descalificării sau revocării la discreţia noastră.\n\nEase nu va fi răspunzător pentru eșecul nici unei Recompense sau Recomandări de a fi emise sau răscumpărate cu exactitate din orice motiv, inclusiv, dar fără a se limita la, orice defecţiune tehnică sau alte probleme legate de orice reţea, sistem informatic, servere, furnizori de acces, echipamente computerizate sau software.\n\n5. DIVERSE\nOrice litigii legate de acești Termeni vor fi soluţionate în conformitate cu termenii de soluţionare a litigiilor din Termenii și condiţiile Ease. Dacă vreo prevedere a acestor Termeni este, din orice motiv, considerată ca fiind invalidă sau neaplicabilă, celelalte dispoziţii ale acestor Termeni vor rămâne executorii, iar dispoziţiile invalide sau neexecutabile vor fi considerate modificate astfel încât să fie valabile și executorii în măsura maximă permisă de lege. Orice renunţare sau neîndeplinire a oricărei prevederi a acestor Termeni cu o singură ocazie nu va fi considerată renunţare la nici o altă dispoziţie sau la o asemenea dispoziţie cu altă ocazie. Nu puteţi atribui niciun drept sau obligaţie în conformitate cu acești Termeni, total sau parţial, fără acordul scris al Ease. Ease poate atribui sau transfera în mod liber acești Termeni. Orice atribuire sau transfer cu încălcarea celor menţionate anterior este nulă. Acești Termeni sunt Termenii finali, compleţi și exclusivi între dvs. și Ease cu privire la subiectele de mai sus și înlocuiesc și contopesc orice discuţii între dvs. și Ease cu privire la aceste subiecte. \n\nDacă aveţi întrebări cu privire la Program sau la acești Termeni, vă rugăm să ne contactaţi la support@ease.ro. ",
```

Comment:
Not sure if `\n\n` works the best here. We would have to use `v-html` to have it working, but that something that can cause some xss attacks. That's why it's not recommended. 

What I would suggest here, if it's possible, to have an array of paragraphs and do a v-for in the component. You can use the power of `i18n template`, see documentation :)

Reply:
not sure if I understand. There are hundreds of places where we use this approach. Why is this one different?

Reply:
Ignore it. Looks good